### PR TITLE
Update API grouping

### PR DESCRIPTION
**Pull request by Steven-Chan**
connect #133 
SkygearIO/features#71

### CODE REVIEW

File: skygear/src/androidTest/java/io/skygear/skygear/ContainerUnitTest.java [hunk 1]
```diff
@@ -102,9 +102,9 @@ public void testContainerPrivateDatabase() throws Exception {
                 "user123",
                 "user123@skygear.dev"
         );
-        container.resolveAuthUser(user);
+        container.auth().resolveAuthUser(user);
```
Comment (Steven-Chan) [inline skygear/src/androidTest/java/io/skygear/skygear/ContainerUnitTest.java]: `container.auth` might be better than `container.auth()`. (same for others)

see https://github.com/SkygearIO/skygear-SDK-Android/issues/133

Comment (Steven-Chan) [inline skygear/src/androidTest/java/io/skygear/skygear/ContainerUnitTest.java]: or `container.getAuth()`

Comment (Steven-Chan) [inline skygear/src/androidTest/java/io/skygear/skygear/ContainerUnitTest.java]: For consistency, with `getPrivateDatabase()`, I prefer `container.getAuth()`.

Comment (cheungpat) [inline skygear/src/androidTest/java/io/skygear/skygear/ContainerUnitTest.java]: I am okay with `getAuth` because it is the language convention.

**Comment by cheungpat** [comment]
I am going to take a second look when the pull request is updated for naming of object getters.

**Comment by Steven-Chan** [comment]
@cheungpat updated

**Comment by Steven-Chan** [comment]
@cheungpat updated

**Pull request merged**